src/marks/image.js

```diff
@@ -104,6 +107,19 @@ export class Image extends Mark {
           .call(applyAttr, "preserveAspectRatio", this.preserveAspectRatio)
           .call(applyAttr, "crossorigin", this.crossOrigin)
           .call(applyChannelStyles, this, channels)
+          .attr("transform", R ? (i) => `rotate(${R[i]})` : this.rotate ? `rotate(${this.rotate})` : null)
+          .attr(
+            "transform-origin",
+            R || this.rotate
+              ? X && Y
+                ? (i) => `${X[i]}px ${Y[i]}px`
+                : X
+                ? (i) => `${X[i]}px ${cy}px`
+                : Y
+                ? (i) => `${cx}px ${Y[i]}px`
+                : `${cx}px ${cy}px`
+              : null
+          )
```

Comment:
unfortunate complexity… am I missing a better way?

Reply:
Can you set use transform-origin="center" instead?

Reply:
I tried it and it doesn’t worker. ☹️ So I guess this is the only option… unless you wrap the image in a G element or something, but I bet that’s worse.